.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "buildtools"]
 	path = buildtools
-	url = git://git.wincent.com/buildtools.git
+	url = https://github.com/wincent/buildtools.git
 [submodule "WOPublic"]
 	path = WOPublic
-	url = git://git.wincent.com/WOPublic.git
```

Comment:
Grrrr. Annoying that Git isn't reliable on this host. I have monitoring set-up, believe it or not, but it somehow fails to notice that clone attempts time out. I need better monitoring...